### PR TITLE
Support qwen1.5 in pytorch engine

### PR DESCRIPTION
**Pull request by grimoire**
This PR requires https://github.com/InternLM/lmdeploy/pull/1075

- [x] #1075

### CODE REVIEW

File: lmdeploy/cli/cli.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def list(args):
         if engine == 'pytorch':
             model_names = [
                 'llama', 'llama2', 'internlm', 'internlm2', 'baichuan2',
-                'chatglm2', 'falcon', 'yi', 'mistral'
+                'chatglm2', 'falcon', 'yi', 'mistral', 'qwen'
```
Comment (lvhan028) [inline lmdeploy/cli/cli.py:110]: may change to `qwen1.5`

**Comment by lvhan028** [comment]
What's the mini version of transformers should I use?

**Comment by grimoire** [comment]
`"transformers_version": "4.37.0",`

**Comment by lvhan028** [comment]
Got lots of unexpected warning messages like the followings. 
```
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
        - Avoid using `tokenizers` before the fork if possible
        - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
```
If setting `TOKENIZERS_PARALLELISM` env in __init__.py, is it appropriate?

**Comment by lvhan028** [comment]
@AllentDan please note that qwen1.5 uses the same chat template with qwen1

**Comment by grimoire** [comment]
> If setting TOKENIZERS_PARALLELISM env in init.py, is it appropriate?

The engine should not affect the behavior of the tokenizer.
I want to remove the tokenizer from the Engine, it is added to align with the Turbomind. But I am not should where has it been used.

**Comment by lvhan028** [comment]
I cannot find a reproducible method for the above warnings. Let's get back the warnings when it happens again.
Chatting with turbomind didn't raise such warnings.

**Review by lvhan028 — APPROVED** [review]

File: lmdeploy/pytorch/config.py [hunk 2]
```diff
@@ -114,7 +114,11 @@ def __build_default():
             num_attention_heads = hf_config.num_attention_heads
             num_key_value_heads = getattr(hf_config, 'num_key_value_heads',
                                           num_attention_heads)
-            sliding_window = getattr(hf_config, 'sliding_window', -1)
+            use_sliding_window = getattr(hf_config, 'use_sliding_window', True)
```
[the remark this answers is not in the document]
Comment (grimoire) [reply lmdeploy/pytorch/config.py:117]: line 117 is used to check flag `use_sliding_window` which is different from `sliding_window`

**Review by RunningLeon — APPROVED** [review]
LGTM

**Comment by serser** [comment]
any plan for turbomind support?

**Pull request merged**